### ISSUE TITLE
Tomography should account for readout imperfections

### ISSUE THREAD

**Issue by marcusps** [rigetti/forest-benchmarking#87]
We should not assume readout to be perfect, and we should offer at least 2 possibilities:

1. use the readout error mitigation in PyQuil's operator estimation
2. estimate the POVMs and use them to update the estimation process

Generally 2 can be done in an ad hoc way (pretend that those POVM estimate are error free), but we can also account for that in some reconstruction methods (linear GST, for example).

**Comment by marcusps** [comment]
@joshcombes @kylegulshen Thoughts?

**Comment by joshcombes** [comment]
Good idea. I'd recommend for it not to be on by default. Lets make sure people understand how bad things are by default.
